addons/html_editor/static/src/main/link/link_plugin.js

```diff
@@ -273,6 +273,23 @@ export class LinkPlugin extends Plugin {
                 }
             }
         });
+        // Some elements can be clicked on without the selection being changed
+        // In those cases the overlay closing and opening must be done manually
+        this.addDomListener(this.editable, "click", (ev) => {
```

Comment:
Hello, I think this is not the right place to fix the issue. The root cause is that when clicking some elements, the selection is not changed. Most likely, they prevented default on these elements. This influences all the selection change handlers.

For example, with the current fix, when you select text from `Home` button and the tool bar will popup, then you click the `Administrator` drop down, the toolbar stays open.

IMO, we should check why the selection change is prevented and make the fix there :thinking: 

Also we already have a `click` event listener, so if you still need to do the changes at `click`, you can probably move the code there :)